package.json

```diff
@@ -122,7 +122,7 @@
     "moment": "^2.17.1",
     "ng2-bootstrap": "^1.1.16",
     "ng2-bs3-modal": "^0.10.4",
-    "ng2-openmrs-formentry": "git+https://github.com/AMPATH/ng2-opemmrs-formentry.git#v2.1.6",
+    "ng2-openmrs-formentry": "git+https://github.com/webtrendzs/ng2-opemmrs-formentry.git#ANFE-70",
```

Comment:
now that testing is complete, get rid of this line, then we can do a form entry release when ANFE-70 is merged in the form entry repo